includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -19,226 +22,196 @@ public static function instance() {
 	}
 
 	public function __construct()	{
+		$this->sections = array(
+			'settings' => __( 'Settings', 'woocommerce-pdf-invoices-packing-slips' ),
+			'status'   => __( 'Status', 'woocommerce-pdf-invoices-packing-slips' ),
+			'tools'    => __( 'Tools', 'woocommerce-pdf-invoices-packing-slips' ),
+			'numbers'  => __( 'Numbers', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I thought about that, but because the other tabs have short names it felt strange to me.

Reply:
If you think it's OK, please ignore my comment.

Reply:
Maybe the others could also give their input, I'm open to change.

@MohamadNateqi @dwalkerpriv @Terminator-Barbapapa

Reply:
I think "Numbers" is good. As Alex mentioned, it feels strange around other short names.

Reply:
I prefer "Document Numbers" as its more descriptive. However if "Numbers" is preferable, we can adjust the description to something like this: "This tab has a record of all of the document numbers generated since the last reset (which happens when you set the next Invoice number value in the settings)."